src/openloco/graphics/gfx.cpp

```diff
@@ -41,9 +41,15 @@ namespace openloco::gfx
     constexpr uint32_t g1_count_temporary = 0x1000;
 
     static loco_global<drawpixelinfo_t, 0x0050B884> _screen_dpi;
+    static loco_global<int16_t, 0x0050B894> _ui_width;
+    static loco_global<int16_t, 0x0050B894> _ui_height;
+    static loco_global<uint32_t, 0x0050B8A4> _dirty_blocks_width_2;
+    static loco_global<uint8_t, 0x0050B8AC> _dirty_blocks_columns;
+    static loco_global<uint8_t, 0x0050B8AD> _dirty_blocks_rows;
     static loco_global<drawpixelinfo_t, 0x005233B8> _windowDPI;
 
     static loco_global<g1_element[g1_expected_count::disc + g1_count_temporary + g1_count_objects], 0x9E2424> _g1Elements;
+    static loco_global<uint8_t[7500], 0x00E025C4> _dirty_blocks;
```

Comment:
If you look at SoftwareDrawingEngine you will see we have used `static loco_global<uint8_t[1], 0x00E025C4> _E025C4;` and then accessed it with auto `grid = Grid<uint8_t>(_E025C4, columns, rows);`.